CHANGES.md

```diff
@@ -12,6 +12,7 @@ We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (
 ## [Unreleased]
 ### Changes
 * We are now opting in to Gradle's new stable configuration cache. ([#1591](https://github.com/diffplug/spotless/pull/1591))
+* Support a file name field in license headers. ([#1605](https://github.com/diffplug/spotless/pull/1605))
```

Comment:
Change this to 

```
### Added
* You can now put the filename into a license header template with `$FILE`. (#1605 fixes #1147)
```

And put it in `plugin-gradle/CHANGES` and `plugin-maven/CHANGES`.

Reply:
Fixed a414e93